### PR TITLE
[FLINK-2889] [tests] Apply JMH on LongSerializationSpeedBenchmark class

### PR DESCRIPTION
**Pull request by gallenvara**
JMH is a Java harness for building, running, and analysing nano/micro/milli/macro benchmarks.Use JMH to replace the old micro benchmarks method in order to get much more accurate results.Modify the `LongSerializationSpeedBenchmark` class and move it to `flink-benchmark` module.

### CODE REVIEW

**Comment by gallenvara** [comment]
JMH result:
longserializationspeed

File: flink-benchmark/src/test/java/org/apache/flink/benchmark/core/memory/LongSerializationSpeedBenchmark.java [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+				.include(LongSerializationSpeedBenchmark.class.getSimpleName())
+				.warmupIterations(2)
+				.measurementIterations(2)
+				.forks(1)
+				.build();
+		new Runner(opt).run();
+	}
+}
+
+
```
Comment (fhueske) [inline flink-benchmark/src/test/java/org/apache/flink/benchmark/core/memory/LongSerializationSpeedBenchmark.java]: Can you remove the empty lines at the end of this file

**Comment by fhueske** [comment]
Thanks for the PR @gallenvara!
Looks good except for the empty lines.

Thanks!

**Comment by fhueske** [comment]
Thanks for the update. 
Looks good to merge.

**Comment by fhueske** [comment]
I'm going to merge this PR.

**Pull request closed without merging**